packages/react/src/components/Tabs/Tabs.tsx

```diff
@@ -122,7 +122,6 @@ const Tabs = ({
       tabIndex: index === activeIndex ? 0 : -1,
       ['aria-controls']: target.current?.id,
       ['aria-selected']: selected,
-      ref: index === activeIndex ? focusedTabRef : null,
```

Comment:
This was clobbering the ref, in reality we don't really need it when we can utilize `[aria-selected="true"]`

Reply:
I believe what you have implemented here should work. However, I have found relying on query selectors in React components to be somewhat finicky. For example, sometimes the DOM isn't in the state you expect it to be in. Or, the structure changes and your query selectors are now incorrect.

fwiw, it should be possible to get this working closer to the way it was originally implemented. See: https://github.com/facebook/react/issues/8873#issuecomment-275423780 and https://github.com/facebook/react/issues/8873#issuecomment-489579878.

```suggestion
      ref: index === activeIndex ? (node) => {
        focusedTabRef.current = node;
        if (typeof child.ref === 'function') {
          child.ref(node)
        } else if (child.ref !== null && child.ref.hasOwnProperty('current')) {
          child.ref.current = node
        }
      } : child.ref
```

Reply:
The query selector shouldn't need to change unless something were to change with the WCAG spec respective of how tabs are implemented. I feel that it's much safer to rely on query here rather than trying to patch into whatever React uses for refs.

Reply:
Additionally, we don't need to clone the ref for every item in the list - we only need the `[aria-selected="true"]` one and the query does exactly that.